frameworks/digital-outcomes-and-specialists/questions/services/lot.yml

```diff
@@ -4,12 +4,24 @@ options:
   -
     label: Digital outcomes
     value: digital-outcomes
+    description: >
+      Research, test, design, build, release, iterate, support or retire a digital service.
+      Examples of digital outcomes include a discovery phase or an online billing application.
   -
     label: Digital specialists
     value: digital-specialists
-  - 
+    description: >
+      Provide specialists to work on a digital service, programme or project. This work must have
+      a defined scope and deliverables. Examples of digital specialists include content designers or
+      technical architects.
+  -
     label: User research studios
     value: user-research-studios
+    description: >
+      Add studios for user research sessions and workshops.
```

Comment:
This is two paragraphs in the prototype. Might be worth asking @ralph-hawkins how much we care about this because it’ll be extra work to markdownify the template.